wallet-webapps-common/src/main/webapp/vue-app/components/SendTokensForm.vue

```diff
@@ -402,6 +402,7 @@ export default {
       );
 
       this.$emit('close');
+      this.showAlert('success',this.$t('exoplatform.wallet.metamask.message.transactionRequested'));
```

Comment:
As discussed with you and @mzorai, this message should be: `The transaction has been sent to Metamask to be signed and sent. Please verify your Metamask.`. The alert type has to be 'info' and should remain until the user interacts with Metamask (accept/reject transaction).
When the user signs the transaction and sends it into blockchain, we should display a Snackbar notification with success message + transaction link.
If the user rejects the transaction or a Metamask failure occurs while sending transaction, a snackbar alert has to be displayed with 'error' type notifying the user that something went wrong. Consequently, the Drawer shouldn't be closed, else the user will loose his message & title.
I let you update this task consequently to be more precise about the expected behavior and change the flow.